suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -34,11 +35,11 @@ const getAccountTypeFromDescriptor = (
 export const importAccountThunk = createThunk(
     `${actionPrefix}/importAccountThunk`,
     ({ accountInfo, accountLabel, coin }: ImportAssetThunkPayload, { dispatch, getState }) => {
-        const device = selectDeviceById(HIDDEN_DEVICE_ID)(getState());
+        const device = selectDeviceById(getState(), HIDDEN_DEVICE_ID);
         const deviceState = HIDDEN_DEVICE_STATE;
 
         if (!device) {
```

Comment:
What happen if you kill app after adding account? Since device reducer is not persisted it will be forgotten and device reducer will be empty unless you import another account?